src/python/WMComponent/TaskArchiver/CleanCouchPoller.py

```diff
@@ -928,19 +908,11 @@ def publishRecoPerfToDashBoard(self, workload):
         logging.debug("Those datasets are interesting %s", str(interestingDatasets))
         if len(interestingDatasets) == 0:
             return
+
         # Request will be only interesting for performance if it's a ReReco or PromptReco
-        (isReReco, isPromptReco) = (False, False)
-        if workload.getRequestType() == 'ReReco':
-            isReReco = True
-        # Yes, few people like magic strings, but have a look at :
-        # https://github.com/dmwm/T0/blob/master/src/python/T0/RunConfig/RunConfigAPI.py#L718
-        # Might be safe enough
-        # FIXME: in TaskArchiver, add a test to make sure that the dataset makes sense (procDataset ~= /a/ERA-PromptReco-vVERSON/DQM)
-        if re.search('PromptReco_', workload.name()):
```

Comment:
I don't think we ever used them. They were supposed to be used to determine future job splitting.